apps/www/__registry__/index.tsx

```diff
@@ -11,7 +11,7 @@ export const Index: Record<string, any> = {
       type: "registry:ui",
       registryDependencies: undefined,
       files: [{
-        path: "/Users/shadcn/Code/shadcn/ui/apps/www/registry/new-york/ui/accordion.tsx",
+        path: "/Users/antoinefamibelle/Develop/shadcn-ui/apps/www/registry/new-york/ui/accordion.tsx",
```

Comment:
careful, seems your `pnpm build:registry` mightve changed some unintended things in index.tsx here

Reply:
In fact ! I pushed a fix sorry for the incovenience !

Reply:
Yes my bad I've removed it from the commit completly sorry for the inconvenience. 👍